nifi-frontend/src/main/frontend/apps/nifi/src/app/pages/flow-designer/ui/canvas/items/processor/edit-processor/edit-processor.component.html

```diff
@@ -17,17 +17,60 @@
 
 <h2 mat-dialog-title>
     <div class="flex justify-between items-baseline">
-        <div>
-            {{ readonly ? 'Processor Details' : 'Edit Processor' }}
+        <div class="flex items-baseline">
+            <div class="mr-2">
+                @if (isInvalid(request.entity)) {
+                    <i
+                        class="mr-2 fa fa-warning caution-color"
+                        nifiTooltip
+                        [tooltipComponentType]="ValidationErrorsTip"
+                        [tooltipInputData]="getValidationErrorsTipData(request.entity)"></i>
```

Comment:
Adding the icon here looks nice, but it seems disjoint with the new start/stop feature since it shows the current status in the icon. Maybe if we hide the start/stop button if the processor is invalid. The disabled "Invalid" button just looks weird in this case:
<img width="1029" alt="Screenshot 2024-11-22 at 9 12 41 AM" src="https://github.com/user-attachments/assets/0773eccd-b2e1-4595-93e9-d004be5784b0">

Reply:
Removed validation errors tooltip and moved the icon back next to the run status.

Reply:
<img width="1031" alt="Screenshot 2024-11-25 at 11 57 05 AM" src="https://github.com/user-attachments/assets/75b544cb-7dce-4381-bf75-dd9b871ae74a">